src/functions_framework/__init__.py

```diff
@@ -45,6 +46,8 @@
 
 _CLOUDEVENT_MIME_TYPE = "application/cloudevents+json"
 
+CloudEventFunction = Callable[[CloudEvent], None]
+HTTPFunction = Callable[[flask.Request], flask.typing.ResponseReturnValue]
```

Comment:
The signature of the HTTP function is a bit verbose :/

Also, if the type checking fails, the error message is ... interesting :)

```
$ cat main.py
import functions_framework
import flask


@functions_framework.http
def hello(request: flask.Request) -> flask.typing.ResponseReturnValue:
    return False
```

```
$ mypy --strict main.py 
main.py:7: error: Incompatible return value type (got "bool", expected "Union[Union[Response, str, bytes, List[Any], Mapping[str, Any], Iterator[str], Iterator[bytes]], Tuple[Union[Response, str, bytes, List[Any], Mapping[str, Any], Iterator[str], Iterator[bytes]], Union[Headers, Mapping[str, Union[str, List[str], Tuple[str, ...]]], Sequence[Tuple[str, Union[str, List[str], Tuple[str, ...]]]]]], Tuple[Union[Response, str, bytes, List[Any], Mapping[str, Any], Iterator[str], Iterator[bytes]], int], Tuple[Union[Response, str, bytes, List[Any], Mapping[str, Any], Iterator[str], Iterator[bytes]], int, Union[Headers, Mapping[str, Union[str, List[str], Tuple[str, ...]]], Sequence[Tuple[str, Union[str, List[str], Tuple[str, ...]]]]]], Callable[[Dict[str, Any], StartResponse], Iterable[bytes]]]")  [return-value]
Found 1 error in 1 file (checked 1 source file)
```

I reckon it's more a Flask issue, would you have an idea how to make that better?

Reply:
Is `return False` something that we'd expect to work? Testing with the latest version of ff I think an error is issued when attempting to return a bool.

Reply:
I don't remember exactly why I returned False in this example, I think the idea was just to return a non-valid type to see what the type checker would show in this case.

So basically, if a developer returns a non-valid type from a `@functions_framework.http` handler, this will be the error show to the developer.